yarn.lock

```diff
@@ -3012,6 +3012,11 @@
   resolved "https://registry.yarnpkg.com/@metamask/contract-metadata/-/contract-metadata-1.23.0.tgz#c70be7f3eaeeb791651ce793b7cdc230e9780b18"
   integrity sha512-oTUqL9dtXtbng60DZMRsBmZ5HiOUUfEsZjuswOJ0yHO24YsW0ktCcgCJVYPv1HcOsF0SVrRtG4rtrvOl4nY+HA==
 
+"@metamask/contract-metadata@1.25.0":
```

Comment:
`@metamask/contract-metadata@1.23.0` & `@metamask/contract-metadata@1.25.0` are co-existing. Worth a check :)

Reply:
thanks! indeed, will fix it